src/main/java/org/mdbenefits/app/preparers/HouseholdExpensesPreparer.java

```diff
@@ -59,9 +59,16 @@ public Map<String, SubmissionField> prepareSubmissionFields(Submission submissio
                 var dependentCareExpensesAmount = (String) submission.getInputData().getOrDefault("expensesDependentCare", "");
                 results.put("expensesDependentCare", new SingleField("expensesDependentCare", dependentCareExpensesAmount + " /month", null));
 
+            } else {
+                results.put("hasDependentCareExpenses", new SingleField("hasDependentCareExpenses", "", null));
             }
         }
 
+        var expensesChildSupport = (String) submission.getInputData().getOrDefault("expensesChildSupport", "");
```

Comment:
Can you note this in the description of the PR as well? Looks like this is adding the PDF fields for the child and dependent expenses.

Reply:
Yes. Does this suffice?

Added logic to append "/month" to child support expenses preparer

Reply:
Oh, that was already in the description, wasn't it?  Sorry, was just going by the title and switching context a bit today.